src/ngResource/resource.js

```diff
@@ -575,7 +575,11 @@ angular.module('ngResource', ['ng']).
 
         forEach(actions, function(action, name) {
           var hasBody = /^(POST|PUT|PATCH)$/i.test(action.method);
-          var cancellable = false;
+          var cancellable = angular.isDefined(action.cancellable) ? action.cancellable :
+            (options && angular.isDefined(options.cancellable)) ? options.cancellable :
+              provider.defaults.cancellable;
+
+          var simulatedTimeout;
```

Comment:
I was trying to find a different name, not to confuse with timeout which is passed to $http. There are quite a few variables that deal with $http timeout.

So it for me it means "non-$http" timeout.